lib/spack/spack/build_systems/nmake.py

```diff
@@ -30,73 +30,117 @@ class NMakePackage(spack.package_base.PackageBase):
 @spack.builder.builder("nmake")
 class NMakeBuilder(BaseBuilder):
     """The NMake builder encodes the most common way of building software with
-    NMake on Windows. It has three phases that can be overridden, if need be:
+    Mircosoft's MSBuild tool. It has three phases that can be overridden, if need be:
```

Comment:
(question) One thing that confuses me: we have an `MSBuild` class as well (i.e. the other file added in this PR): is that a distinct concept from Microsoft's MSBuild tool?